tests/expectations/metrics/test_metric_providers.py

```diff
@@ -29,17 +34,27 @@
 from great_expectations.expectations.metrics.table_metric_provider import (
     TableMetricProvider,
 )
-from great_expectations.expectations.registry import (
-    _registered_metrics,
-)
+from great_expectations.expectations import registry
 
 
-def test__base_metric_provider__registration():
-    """This tests whether the MetricProvider class registers the correct metrics."""
+@pytest.fixture
+def mock_registry(monkeypatch: pytest.MonkeyPatch):
+    # ensures consistent state among test cases runs
```

Comment:
```suggestion
    """Ensures consistent `_registered_metrics` state among test cases runs."""
```
Might as well make this a docstring